core-bundle/package.json

```diff
@@ -0,0 +1,17 @@
+{
+    "private": true,
+    "scripts": {
+        "dev": "NODE_ENV=development node_modules/.bin/encore dev --watch",
+        "prod": "env NODE_ENV=production node_modules/.bin/encore production"
+    },
+    "devDependencies": {
+        "@babel/core": "^7.19.3",
+        "@babel/preset-env": "^7.19.4",
+        "@hotwired/stimulus": "^3.1.0",
+        "@hotwired/stimulus-webpack-helpers": "^1.0.1",
+        "@symfony/webpack-encore": "^4.0.0",
+        "autoprefixer": "^10.4.12",
+        "webpack": "^5.74.0",
+        "webpack-cli": "^4.10.0"
+    }
```

Comment:
IMHO we should add a `browserslist` section here (or add an additional file for that):

Something like this (but with our actual requirements that I'm unsure about):

```json
"browserslist": [
    "> 0.5%",
    "not dead",
    "not op_mini all",
    "not chrome < 51",
    "not safari < 10"
  ]
```

Reply:
added in 42ad6f0578ce726dbdb405bd36c25c98df993e65 according to https://symfony.com/doc/current/frontend/encore/postcss.html#adding-browserslist-to-package-json